src/Provider/PickPoint/Tests/IntegrationTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Geocoder package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Geocoder\Provider\PickPoint\Tests;
+
+use Geocoder\IntegrationTest\ProviderIntegrationTest;
+use Geocoder\Provider\PickPoint\PickPoint;
+use Http\Client\HttpClient;
+
+/**
+ * @author Vladimir Kalinkin <vova.kalinkin@gmail.com>
+ */
+class IntegrationTest extends ProviderIntegrationTest
+{
+    protected $testAddress = true;
+    protected $testReverse = true;
```

Comment:
This should be the other way around. Look in `ProviderIntegrationTest`. All tests are enabled by default, you should write ie `$testIpv4 = false`